subprojects/plugin-use/src/integTest/groovy/org/gradle/plugin/repository/ResolvingFromSingleCustomPluginRepositorySpec.groovy

```diff
@@ -23,6 +23,7 @@ import org.gradle.test.fixtures.maven.MavenFileRepository
 import org.gradle.test.fixtures.plugin.PluginBuilder
 import org.gradle.util.Requires
 import org.gradle.util.TestPrecondition
+import spock.lang.IgnoreRest
 import spock.lang.Unroll
 
```

Comment:
We are missing sad day cases here, like proper error messages when a plugin can't be resolved.